src/Twig/Components/SidebarSubscriptionComponent.php

```diff
@@ -25,22 +26,22 @@ class SidebarSubscriptionComponent
 
     public ?string $sort;
 
+    public function __construct(private readonly MagazineRepository $magazineRepository)
+    {
+    }
+
     #[PostMount]
     public function PostMount(): void
     {
         $max = 50;
         $this->magazines = [];
-        foreach ($this->user->subscriptions as /* @type MagazineSubscription $sub */ $sub) {
-            $this->magazines[] = $sub->magazine;
-        }
         if (ThemeSettingsController::ALPHABETICALLY === $this->sort) {
-            usort($this->magazines, fn ($a, $b) => $a->name > $b->name ? 1 : -1);
+            $this->magazines = $this->magazineRepository->findMagazineSubscriptionsOfUser($this->user, SubscriptionSort::Alphabetically, $max);
         } else {
-            usort($this->magazines, fn ($a, $b) => $a->lastActive < $b->lastActive ? 1 : -1);
+            $this->magazines = $this->magazineRepository->findMagazineSubscriptionsOfUser($this->user, SubscriptionSort::LastActive, $max);
         }
-        if (\sizeof($this->magazines) > $max) {
+        if (\sizeof($this->magazines) === $max) {
```

Comment:
sorry equal to max?

Reply:
maybe `>=` ?

Reply:
is there a type check in PHP like `>==` xD?

Reply:
The reason is, the max is 50, so the array can never be bigger than 50. Before the change the array was not limited in size

Reply:
Yea ok..